src/vis_cpu/wrapper.py

```diff
@@ -61,36 +70,45 @@ def simulate_vis(
     latitude : float, optional
         The latitude of the center of the array, in radians. The default is the
         HERA latitude = -30.7215 * pi / 180.
+    beam_spline_opts : dict, optional
+        Options to be passed to :meth:`pyuvdata.uvbeam.UVBeam.interp` as `spline_opts`.
 
     Returns
     -------
     vis : array_like
-        Complex array of shape (NAXES, NFEED, NFREQS, NTIMES, NANTS, NANTS)
+        Complex array of shape (NFREQS, NTIMES, NFEED, NFEED, NANTS, NANTS)
```

Comment:
Note API breaking change

Reply:
(IIRC we chose this ordering to match pyuvdata array ordering)

Reply:
Noted in the changelog. I don't think the old way matched pyuvdata -- pyuvdata's ordering has `Nblts` as the first axis, which matches neither of these. The new ordering is more simple given the way we loop.